EndlessClient/UIControls/CreateCharacterControl.cs

```diff
@@ -68,6 +74,13 @@ public void NextHairColor()
             RenderProperties = RenderProperties.WithHairColor((RenderProperties.HairColor + 1) % 10);
         }
 
+        public void NextDirection()
```

Comment:
Is this method invoked anywhere? My limited "Ctrl+F" PR review search ability isn't turning up anything. If it's not used, please delete